src/libostree/ostree-bootloader-grub2.c

```diff
@@ -357,15 +358,11 @@ _ostree_bootloader_grub2_write_config (OstreeBootloader      *bootloader,
   if (use_system_grub2_mkconfig && ostree_sysroot_get_booted_deployment (self->sysroot) == NULL
       && g_file_has_parent (self->sysroot->path, NULL))
     {
-      g_autoptr(GPtrArray) deployments = NULL;
       OstreeDeployment *tool_deployment;
       g_autoptr(GFile) tool_deployment_root = NULL;
 
-      deployments = ostree_sysroot_get_deployments (self->sysroot);
-
-      g_assert_cmpint (deployments->len, >, 0);
-
-      tool_deployment = deployments->pdata[0];
+      g_assert_cmpint (new_deployments->len, > , 0);
```

Comment:
Minor/nit: extra space between `>` and comma.

Reply:
Ahh thanks, fixing that now!

Reply:
Fixed!